src/Microsoft.OData.Client/ALinq/UriHelper.cs

```diff
@@ -158,6 +173,24 @@ internal static class UriHelper
         /// <summary>The encoded { sign</summary>
         internal const string ENCODEDBRACESIGN = "%7B";
 
+        /// <summary>average</summary>
+        internal const string AVERAGE = "average";
+
+        /// <summary>sum</summary>
+        internal const string SUM = "sum";
+
+        /// <summary>min</summary>
+        internal const string MIN = "min";
+
+        /// <summary>max</summary>
+        internal const string MAX = "max";
+
+        /// <summary>countdistinct</summary>
+        internal const string COUNTDISTINCT = "countdistinct";
+
+        /// <summary>$count</summary>
+        internal const string VIRTUALPROPERTYCOUNT = "$count";

```

Comment:
>VIRTUALPROPERTYCOUNT [](start = 30, length = 20)

why has a name "virtual property count" equals "$count"?

Reply:
@xuzhg I borrowed this one from ODL https://github.com/OData/odata.net/blob/576c4ab0337bb1cecda85f4199bcc406d55b787b/src/Microsoft.OData.Core/UriParser/Aggregation/AggregationMethod.cs#L32